<launch>
  <node pkg="gimbal_control" name="gimbal_control_node" type="gimbal_control_node" output="screen">
  <param name="pid_control_flag" type="bool" value="true"/>
  
  <param name="p_pitch" type="double" value="800.0"/>
  <param name="i_pitch" type="double" value="0.0"/>
  <param name="d_pitch" type="double" value="0.0"/>
  <param name="constrain_pitch" type="double" value="300.0"/>
  <param name="start_ditstance_pitch" type="double" value="0.2"/>
  <param name="p_yaw" type="double" value="800.0"/>
  <param name="i_yaw" type="double" value="0.0"/>
  <param name="d_yaw" type="double" value="0.0"/>
  <param name="constrain_yaw" type="double" value="300.0"/>
  <param name="start_ditstance_yaw" type="double" value="0.2"/>
  
  <param name="body_min_pitch" type="double" value="-80.0"/>
  <param name="body_max_pitch" type="double" value="-10.0"/>
  <param name="body_min_yaw" type="double" value="-45.0"/>
  <param name="body_max_yaw" type="double" value="45.0"/>
  
  
  </node>
</launch>
